Havit.Blazor.Components.Web.Bootstrap/Navigation/HxSidebar.razor

```diff
@@ -47,9 +48,11 @@
 
             @if (FooterTemplate is not null)
             {
-                <div class="hx-sidebar-footer">
-                    @FooterTemplate
-                </div>
+	            <div class="hx-sidebar-footer">
+		            <CascadingValue Value="@this">
+			            @FooterTemplate
+		            </CascadingValue>
+	            </div>
```

Comment:
We are aware of the original issue with the FooterTemplate, but it should be resolved another way.

We decided to provider Context to the FooterTemplate RenderFragment where the RenderFragment will be able to provide different content when the sidebar is collapsed.
Related issue: #272 (waiting for implementation)
Similar PR: #267 (we already added the context on some places)

Please revert this solution.
If you want to resolve the footer issue, take the #272 and create a separate PR for that.

Reply:
@Xopabyteh Reverted those footer-related changes to help you focus on the original request - demos. We can work on the footer in separate PR.